frappe/desk/form/assign_to.py

```diff
@@ -71,7 +72,9 @@ def add(args=None, *, ignore_permissions=False):
 		else:
 			from frappe.utils import nowdate
 
-			if not args.get("description"):
+			description = str(args.get("description", ""))
```

Comment:
There could be the edge case of `args = {"description": None}` but how likely is it really :smile: